addons/mail/static/tests/qunit_suite_tests/utils/throttle_tests.js

```diff
@@ -137,18 +119,15 @@ QUnit.test('throttled call reinvocation', async function (assert) {
     });
 
     let funcCalledAmount = 0;
-    const throttledFunc = throttle(
-        messaging,
-        () => {
+    const throttledFunc = messaging.models['Throttle'].insert({
+        func: () => {
             funcCalledAmount++;
             return `func_result_${funcCalledAmount}`;
         },
-        1000,
-        { silentCancelationErrors: false }
-    );
-    this.throttles.push(throttledFunc);
-
-    throttledFunc().then(result => {
+        hasSilentCancelationErrors: false,
```

Comment:
useless feature only used in tests?

Reply:
Yes, I kept it just to make sure the conversion to models show the exact same behaviour that was covered in the tests.
I was thinking that we could remove this field, but at a glance the tests were still covering meaningful specs of Throttle, and it just happens that intercepting the errors make it easier to detect proper behaviour when canceling/flushing a pending throttle.